packages/rocketchat-api/server/helpers/getUserInfo.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'underscore';
```

Comment:
@sampaiodiego can correct me if I'm wrong, but I believe we are wanting to lower our relienace on underscore. So, the `_.pick` below should be pretty easy to change. :)

Reply:
yes.. avoid it as much as we can =)

Reply:
I can change @graywolf336. ;)